### ISSUE TITLE
Ditch RPM for FPM

### ISSUE THREAD

**Issue by ancsik** [sherl0k/InstallTheGroove#1]
https://github.com/jordansissel/fpm/wiki

You get .deb for free on top of the .rpm and help free users from unnecessary degrees of vendor lock (Both RHELvsDeb and potentially InstTG vs. Using the packages on some other image depending on details). The dependency list will be the same (excepting slight naming differences between the two management systems). You also swap RPM .spec files (a sickening nightmare) and all of the horrible design choices behind RPM's package building in favor of "It's that directory's files and this bash script will tell you what to do with them", no .spec or anything.

It's going to be easier and more welcoming to the existing install base (stock boxors are some old-ass debian, all of the boxorroxor guides are debian/ubuntu) to have free .deb support, if nothing else.

RPM is old and the spec format is garbage.

**Comment by sherl0k** [comment]
oh this looks like fun. being able to build for deb is a plus. will definitely implement.

**Comment by ancsik** [comment]
Yeah, I learned about FPM because my last company's build environment could only be modified by .rpm as a stability requirement, but we needed to add Gradle, which isn't actively supported by any package manager (On the Ubuntu side, the package is like 2 years out of date, since it's easier to just download the tar and call the executable directly [add it to your $PATH, if nothing else]). 30 minutes later, the build team had no clue how I made them a fresh .rpm out of thin air.

**Comment by terabyte** [comment]
+1, this looks great.

Also, as a contributor to the openitg project (and thus a person owed some ire for letting it wither quite a bit), I just wanted to thank you.  This is _awesome_ and I think I'm going to try it for my current project (restoring an old boxor whose mobo died and thus needs a newer linux install).

I'm working on building openitg itself using docker containers, so a future option for installthegroove may be to support using original OpenITG binaries instead of SM5 (but if SM5 supports piuio, there may not be a strong reason to do that anymore anyways...)
